### ISSUE TITLE
Vérifier différents types de phénomènes de vigilance météo et iconographie associée

### ISSUE THREAD

**Issue by fcoufour** [betagouv/recosante#167]
J'ai vu qu'il manquait 2 phénomènes plutôt estivaux côté front-end (Canicule, Vagues-Submersion) sur un total de 9 phénomènes d'après la documentation technique de Météo France. A voir si c'est volontaire ou non.

Aussi, les icônes sont des fichiers PNG créés à partir de GIF de mauvaise qualité et de tailles disparates ce qui se voit à l'écran.
Est-on en attente des fichiers sources ou doit-on les vectoriser ?
A-t-on l'accord du fournisseur pour utiliser ces images ?

Il semble y avoir eu 3 petites erreurs côté frontend sur la correspondance entre icône et phénomène : Crues, Neige, Grand Froid. A vérifier ensemble car certaines appellations diffèrent un peu entre site MF et doc technique : Vent vs Vent violent, Neige vs Neige-verglas

<img width="727" alt="phenomenes" src="https://user-images.githubusercontent.com/23194091/147657466-95108d48-6a50-41f1-beac-c0cd6eca8f0e.png">

Pour finir, il existe 2 versions de Vigilance Météo proposé par MF :
http://vigilance2019.meteofrance.com/
https://vigilance.meteofrance.fr/
Laquelle est notre référence ?

**Comment by florianpanchout** [comment]
Pour les png dégueulasses, je suis en attente d'une réponse (https://github.com/betagouv/recosante/issues/144#issuecomment-998621862). (mais j'ai de gros doutes sur le fait que l'on récupère des svg...)

Je ne suis pas sur de comprendre comment fonctionne l'affichage des phénomènes. L'api me renvoie la liste (que je pensais exhaustive) des phénomènes, et je ne me suis basé la dessus. Mais j'imagine qu'elle ne me ne renvoie au final que ceux qui peuvent se produire pour la commune ? (@l-vincent-l ?)

**Comment by fcoufour** [comment]
Je peux répondre en partie pour Vincent. L'API renvoie tous les épisodes de vigilance qui concerne la commune sélectionnée (niveau départemental) au moment de la consultation donc il ne s'agit bien que d'un échantillon. Exemple extrême, il y a des chances infimes qu'on puisse avoir une jour une Canicule pendant une période de Grand Froid. Les 9 phénomènes figurent bien dans l'infobulle côté frontend par contre : `La vigilance couvre 9 phénomènes : vent violent, vagues-submersion, pluie-inondation, crues, orages, neige-verglas, avalanches, canicule et grand froid.`.

Nous sommes le 29/12 au moment où j'écrit et depuis ce matin on a une vigilance orange et une vigilance jaune dans les "Alpes" ce qui me paraît troublant. Peut-être que c'est dû à une gestion particulière de notre côté pour avoir une vision journalière alors qu'elle est horaire chez MF mais pas certain que ce soit le bon choix UX. A discuter mais 3 possibilités : soit on met les 2 et on explique pourquoi en apportant du détail, soit on met seulement la plus grave du jour, soit on met celle du moment (horaire de consultation mais quid du contenu de l'email).

**Comment by florianpanchout** [comment]
Pour la partie "affichage jaune + orange" Il me semble qu'il avait été décidé ici (https://github.com/betagouv/recosante/issues/144) de n'afficher que la vigilance la plus haute de la journée pour l'instant. Je m'attendais juste à ce que la sélection se fasse au niveau de l'api et non du front.

**Comment by florianpanchout** [comment]
J'ai ajouté les phénomènes manquants (et corrigés certaines icônes).

Pour l'appellation des phénomènes, je me suis basé de ce que je récupère de l'api ("Neige" au lieu de "Neige Verglas", etc.)

Il reste toujours la question des png de mauvaise qualité. @annepoirot est ce qu'on a une réponse de Météo France la dessus ?

**Comment by annepoirot** [comment]
Je pense qu'il faut se baser sur ce que météo france fait en front pour les appellations. 

Concernant les png, je vais les contacter mais je pense qu'ils vont directement aller voir ce qu'on fait et du coup faut peut-être qu'on soit au clair sur l'affichage de la vigilance (car aujourd'hui par exemple c'est différent de météo france et j'ai peur que ça nous retombe dessus)

**Comment by fcoufour** [comment]
D'accord avec Anne,"Neige-verglas"
https://vigilance.meteofrance.fr/fr/neige-verglas

**Comment by fcoufour** [comment]
Exemple où on affiche une information différente de MF car ils utilisent une fenêtre glissante (vigilance jaune dans la nuit de mardi) :

- https://recosante.beta.gouv.fr/place/64122/biarritz/
<img width="601" alt="biarritz-vigilance-meteo-20220103" src="https://user-images.githubusercontent.com/23194091/147922942-06c75f14-3022-4a51-a8d2-9ff9c376ba2b.png">

- https://vigilance.meteofrance.fr/fr/pyrenees-atlantiques
<img width="627" alt="pyrenees-atlantique-mf-20220103" src="https://user-images.githubusercontent.com/23194091/147922970-037eefac-46d5-4eaa-997d-dcb6fc2089d8.png">

La règle métier est donc à ajuster pour être conforme à MF.

**Comment by florianpanchout** [comment]
@annepoirot Est ce que c'est déjà le cas pour l'indice ATMO ? L'indice est valide de 6h à 6h ou de minuit à minuit ? (l'api me parle d'une validité de minuit à minuit).

Parce que si nos deux indicateurs quotidiens ne sont pas sur la même temporalité, ça risque de créer une incompréhension

**Comment by fcoufour** [comment]
J'ai regardé ce qu'il y a en base et la plupart du temps ce sont des informations journalières de minuit à minuit. Les dates sont issues des différentes APIs régionales (par ex. https://github.com/betagouv/indice_pollution/blob/master/indice_pollution/regions/Normandie.py#L121).
On peut repasser les sources au peigne fin mais Vincent doit avoir tout cela en tête.

OK avec toi sur le fait que ça peut être confus pour l'utilisateur mais ce serait encore plus confus d'afficher des données différentes du site de MF. Les petites lignes peuvent être utilisées pour donner plus de contexte.

**Comment by annepoirot** [comment]
A chaque publication de données pour l'indice ATMO, sont publiées les données J-1, J+1 et parfois J+2. Les données sont actualisées au maximum à 14h tous les jours. Donc quand on envoie la NL le matin, on partage des données de prévision publiées la veille. C'est données sont valables jusqu'à minuit le jour même, mais peuvent potentiellement être actualisées à 14h si l'indice du jour est en réalité différent de l'indice de prévision modélisé la veille. 

Par exemple pour Auvergne Rhone Alpes aujourd'hui: 
<img width="898" alt="Capture d’écran 2022-01-03 à 14 53 07" src="https://user-images.githubusercontent.com/63290799/147938748-0db9791c-9f22-4e5f-a9ac-c35039f232db.png">

Effectivement le problème c'est la différence d'affichage entre notre tableau de bord et la source...

**Comment by florianpanchout** [comment]
@fcoufour Yep, il manque toujours la date de validité de la vigilance globale d'ailleurs (https://github.com/betagouv/recosante/issues/144#issuecomment-998764451). Qui pour l'instant n'est donc pas affichée en dessous de la tuile.

**Comment by fcoufour** [comment]
@annepoirot D'où est issu ton screenshot ?

**Comment by annepoirot** [comment]
Pour les appellation des phénomènes : on modifie les termes dans l'API 
"Neige" => "Neige-verglas" https://vigilance.meteofrance.fr/fr/neige-verglas
"Vent" => "Vent violent" https://vigilance.meteofrance.fr/fr/vent-violent

Il faut aussi modifier l'historique pour qu'il soit propre ainsi que les icones.

**Comment by annepoirot** [comment]
> @annepoirot D'où est issu ton screenshot ?

https://data-atmoaura.opendata.arcgis.com/datasets/813d359d4e7e4705af45a6f28294cad9_0/explore?location=45.428977%2C4.621968%2C8.45&showTable=true

**Comment by florianpanchout** [comment]
Les images et couleurs s'affichent maintenant correctement
